packages/camera/camera_avfoundation/example/lib/camera_controller.dart

```diff
@@ -171,26 +171,36 @@ class CameraValue {
 /// outside of the overall example code.
 class CameraController extends ValueNotifier<CameraValue> {
   /// Creates a new camera controller in an uninitialized state.
-  CameraController(
+  /// Deprecated, use [withSettings].
```

Comment:
Why do we have deprecated code in an example app that exists for our own testing?

Reply:
Comment removed.